marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/flowRunner.sjs

```diff
@@ -150,6 +150,11 @@ function processContentWithStep(stepExecutionContext, contentArray, databaseWrit
     return [];
   }
 
+  // if targetCollectionsAdditivity is set, then target documents should inherit collections from source documents
```

Comment:
No need for a comment here - it's only restating what the code is already saying.